### ISSUE TITLE
Glitch with async / await highlighting

### ISSUE THREAD

**Issue by dsl101** [Benvie/JavaScriptNext.tmLanguage#4]
I have a Vue component which has 2 very similar methods that use an asynchronous call and then do something when it completes. Note this code works, but the syntax highlighting doesn't work for the second function. The `await` reserved word is in plain white, not red. I know I could store the return value in the second function before calling `$router.push()`, but I just want to use the return value directly in the argument to that call. It seems the matcher doesn't recognise `await` if it is inside parentheses.
```
async handleCreateEvent (data) {
  await this.createEvent(data)
  this.reloadEventsTrigger = !this.reloadEventsTrigger
},
async handleCreateEventAndOpen (data) {
  this.$router.push('/' + await this.createEvent(data))
},
```
![image](https://user-images.githubusercontent.com/900542/34722428-c92a27f4-f53e-11e7-9043-72207b3d29b9.png)

**Comment by deathlynocturnal** [comment]
Yeah I have seen this, looks like it doesn't recognise the await word when contained in a function call.

**Comment by deathlynocturnal** [comment]
Is this project dead?
